api-auto-registration/key-concepts.hbs.md

```diff
@@ -132,13 +148,98 @@ To use an Ingress instead, your controller reads the URL from the jsonPath speci
 
 ### <a id='status-fields'></a>APIDescriptor Status Fields
 
-When processing an APIDescriptor several fields are added to the `status`. One of these is `conditons`, which provide information useful for troubleshooting. The conditions are explained in the [Troubleshooting Guide](../api-auto-registration/troubleshooting.hbs.md).
+When processing an APIDescriptor several fields are added to the `status`. One of these is `conditons`,
+which provide information useful for troubleshooting. The conditions are explained in the
+[Troubleshooting Guide](../api-auto-registration/troubleshooting.hbs.md).
+
+In addition to `conditions` the `status` contains a couple of other useful fields. The following is
+a list of these fields with a brief explanation of what they contain.
+
+```yaml
+status:
+  registeredEntityURL:    # Url of the corresponding API Entity in Tanzu Developer Portal
+  registeredTapUID:       # Unique identifier for the corresponding API Entity in Tanzu Developer Portal
+  resolvedAPIServerURL:   # Url to the API runtime server
+  resolvedAPISpecURL:     # Url used to retrieve the full API Spec by Api Auto Registration
+  resolvedAPISpec:        # Full API Spec as retrieved by Api Auto Registration
+  resolvedAPISpecHash:    # Hash value of the `resolvedAPISpec`. This field can be used to see whether the spec has been updated or not.
+  APISpecLastUpdateTime:  # Timestamp representing the server time when API Spec was last updated. It is not guaranteed to be set in happens-before order across separate operations. It is represented in RFC3339 form and is in UTC.
+```
+
+## <a id='curated-api-descriptor'></a>CuratedAPIDescriptor Custom Resource Explained
+
+To curate one or more Workload OpenAPI specifications into a single aggregated API,
+you may create a custom resource of type `CuratedAPIDescriptor`.
+The information from this custom resource references a list of APIDescriptors and how they should
+be aggregated together through path-based routing.
+
+If a valid route provider is specified, e.g. `spring-cloud-gateway` for
+[Spring Cloud Gateway for Kubernetes](../spring-cloud-gateway/about.hbs.md) (SCG for short),
+the API Auto Registration controller will find the [SpringCloudGateway (SCG)](https://docs.vmware.com/en/VMware-Spring-Cloud-Gateway-for-Kubernetes/2.1/scg-k8s/GUID-developer-resources-springcloudgateway.html)
+resource and automatically creates the following routing resources for you to expose your curated APIs
+as one:
+
+- [SpringCloudGatewayRouteConfig (SCGRC)](https://docs.vmware.com/en/VMware-Spring-Cloud-Gateway-for-Kubernetes/2.1/scg-k8s/GUID-developer-resources-springcloudgatewayrouteconfig.html):
```

Comment:
I don't know the right answer here but we have been hurt by stale URLs a lot. The old versions getting leftover when we release new versions. Any way to link without having the versions?